server/zanata-frontend/src/app/containers/Admin/index.js

```diff
@@ -92,31 +92,31 @@ class Admin extends Component {
               <Col xs={24} sm={12} lg={6}>
                 <Link id='Admin_Role_Assignment_Rules_home'
                   className='list-group-item'
-                  link='/admin/rolerules' useHref>
+                  link='rolerules' useHref>
                   <Icon name='assign' className='s2' />
                   Role assignment rules
                 </Link>
               </Col>
               <Col xs={24} sm={12} lg={6}>
                 <Link id='Translation_Memory_home'
                   className='list-group-item'
-                  link='/tm' useHref>
+                  link='/zanata/tm' useHref>
```

Comment:
guess this won't work when not running under '/zanata'

Reply:
Thank you. I will close PR. Lets use yours.